tests/FormSpamProtectionExtensionTest.php

```diff
@@ -44,6 +44,7 @@ public function testEnableSpamProtectionThrowsException()
         $this->expectException(LogicException::class);
         $this->expectExceptionMessage('No spam protector has been set. Null is not valid value.');
 
+        Config::modify()->set(FormSpamProtectionExtension::class, 'default_spam_protector', null);
```

Comment:
You should use `Config::withConfig` to avoid your config change bleeding into the next test.

Reply:
It won't bleed over, we use `Config::modify()->set()` in hundreds of existing tests. It resets itself between tests.

Reply:
Tested it locally and it does NOT bleed.